### PR TITLE
add SMPP and secure SMPP port to ping

### PR DESCRIPTION
**Pull request by knosach**
add SMPP ports to ping for issue RestComm/load-balancer#29

### CODE REVIEW

**Comment by knosach** [comment]
@deruelle Can you review my changes?

File: core/src/main/java/org/mobicents/ha/javax/sip/LoadBalancerHeartBeatingServiceImpl.java [hunk 1]
```diff
@@ -575,6 +580,36 @@ protected void updateConnectorsAsSIPNode() {
...
+						}
+					} catch (Exception e) {} //Ignore any exceptions
+
+					if(smppBound && smppPort!=null){
+						sipStackProperties.setProperty(LOCAL_SMPP_PORT, String.valueOf(smppPort));
+					} 
+					if (smppSslBound && smppSslPort!=null){
+		                sipStackProperties.setProperty(LOCAL_SMPP_SSL_PORT, String.valueOf(smppSslPort));
+					}
+				}
```
Comment (deruelle) [inline core/src/main/java/org/mobicents/ha/javax/sip/LoadBalancerHeartBeatingServiceImpl.java]: We don't create any SMPP socket binding group for SMPP in standalone-sip.xml right ? If no, then this part of the code should not be present.

File: core/src/main/java/org/mobicents/ha/javax/sip/MultiNetworkLoadBalancerHeartBeatingServiceImpl.java [hunk 2]
```diff
@@ -452,6 +457,37 @@ protected void updateConnectorsAsSIPNode(SipLoadBalancer loadBalancer) {
...
+					} catch (Exception e) {} //Ignore any exceptions
+
+					if(smppBound && smppPort!=null){
+						sipStackProperties.setProperty(LOCAL_SMPP_PORT, String.valueOf(smppPort));
+					} 
+					if (smppSslBound && smppSslPort!=null){
+		                sipStackProperties.setProperty(LOCAL_SMPP_SSL_PORT, String.valueOf(smppSslPort));
+					}
+				}
+
```
Comment (deruelle) [inline core/src/main/java/org/mobicents/ha/javax/sip/MultiNetworkLoadBalancerHeartBeatingServiceImpl.java]: We don't create any SMPP socket binding group for SMPP in standalone-sip.xml right ? If no, then this part of the code should not be present.

**Comment by deruelle** [comment]
@knosach commented. please review my comments

**Comment by knosach** [comment]
@deruelle I removed code where we have got SMPP ports from socket binding group

**Pull request merged**